firefox-ios/Client/Frontend/Browser/BrowserViewController/Actions/NavigationBrowserAction.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Redux
+
+/// Actions that are related to navigation from the user perspective
```

Comment:
user perspective for now, we can update this comment if we see that there are API actions taken that uses this